README.md

```diff
@@ -326,6 +334,22 @@ document.
 
 ---
 
+```
+DELETE ${hubUrl}/delete/${address}/${path}
+```
+
+This performs a deletion of a file in the gaia hub at `${path}`. 
+
+On success, it returns a `202` status. Returns a `403` if the path does 
```

Comment:
Shouldn't this be a 404 if the path doesn't exist?  Also, 403 implies that the caller isn't authorized to do the action, not that the action is inapplicable to the object.  Should we instead be sending a 422 for the case where the object isn't a file, and a 400 for the case where the path itself is malformed?

Reply:
This is mirroring the putFile responses which is a 403 on BadPathError. The deleteFile method in drivers uses much of the same path validation. 
I agree we should change to those status codes. Should update putFile as well?

Reply:
This should now be fixed throughout the PR (all driver, http response handler, and docs). Can you confirm?

Reply:
Yup, looks good!